core/src/test/java/nu/yona/server/analysis/service/AnalysisEngineServiceTests.java

```diff
@@ -399,7 +403,7 @@ public void noMessagesCreatedOnNonZeroBudgetGoal()
 	{
 		// Execute the analysis engine service.
 		Set<String> conflictCategories = new HashSet<String>(Arrays.asList("social"));
-		service.analyze(new NetworkActivityDTO(userAnonID, conflictCategories, "http://localhost/test"));
+		service.analyze(new NetworkActivityDTO(userAnonID, conflictCategories, "http://localhost/test", Optional.empty()));
```

Comment:
Yes, but since it's just for the test, I don't think we should change the production code.